src/helpers/addError.ts

```diff
@@ -0,0 +1,16 @@
+import { FormContext } from "../Form";
+import { ModelError } from "../Model";
+
+export function addError(context: FormContext, error: any): void {
```

Comment:
Error should have types from Axios.
Install axios as devDependency and use only types from it.